tensordata/scRNA.py

```diff
@@ -0,0 +1,272 @@
+from concurrent.futures import ProcessPoolExecutor
+from pathlib import Path
+
+import anndata
+import numpy as np
+import numpy.typing as npt
+import pandas as pd
+import scanpy as sc
```

Comment:
Since this will be imported in a lot of places, it would be really, really beneficial to remove the scanpy dependency. Scanpy imports a ton of random packages that makes it fragile and heavy. I think we only need `read_10x_mtx()` so perhaps that code can be copied in here?

Reply:
This is done, although I removed some options from the scanpy implementation. If we end up needing these options, I can always add them back.